pytorch_blade/WORKSPACE

```diff
@@ -126,7 +126,7 @@ new_local_repository(
 new_local_repository(
   name = "mlir-hlo",
   build_file = "third_party/torch-mlir/externals/mlir-hlo/BUILD",
-  path = "../tf_community/tensorflow/compiler/xla/mlir_hlo"
+  path = "third_party/torch-mlir/externals/mlir-hlo"
```

Comment:
We need to ensure these two directory contains identical content, do we?
This also means we can't rebase to arbitrary commit of torch-mlir and tf_community?

Reply:
Yes, they both depend on MHLO; we must keep them consistent.

Reply:
Every one who's going to do rebase, pls note this...